### PR TITLE
300 doc pado

### LINKED ISSUE #300: Add PADO to Linea doc in Tooling & Infra section
Hello Team,

I'd like to propose adding a section on the [PADO](https://padolabs.org/) project within "Build on Linea > Tooling and Infrastructure > Oracles". PADO is a cryptography-based infrastructure to bring all Internet data into smart contracts. To be more precise, PADO leverages interactive ZK and MPC techniques to "blindly" validate Internet data via TLS channels. Such techniques can be roughly viewed as “private oracles” that retrieve users’ private data from Internet sources, compared with public oracles that feed public web data to smart contracts. In addition, the Internet data is reliable because the data authenticity is guaranteed under the ZK-based attestation pipeline, and the privacy is well protected regarding to specific application logic by using zero-knowledge proofs.

PADO dApp is developed on Linea, and integrated with Verax attestation framework. Users can create their own Internet data attestations by using PADO dApp. These attestations can be stored and accessed from Verax by all Linea dApps. 

Introducing PADO section in the documentation gives general guidance on how users or developers can create and use off-chain data attestations in various scenarios. 

I'll create a PR for this section. Feedback and thoughts are welcome!

### PR DESCRIPTION
**Pull request by xiafubiao**
Add PADO to doc/tooling and infrastructure/attestation, with a created new category "Attestation", fixes #300

### CODE REVIEW

**Review by emilyJLin95 — CHANGES_REQUESTED** [review]
Hi can you remove the package-lock.json change?

Can you also run `npm run lint` and edit `project-words.txt` to allowlist new words

Also just `npm run build` to make sure everything is building correctly?

**Review by emilyJLin95 — APPROVED** [review]
Can you add these words the `project-words.txt` and run `npm lint` to make sure the lint passes

/home/runner/work/doc.zk-evm/doc.zk-evm/docs/build-on-linea/tooling/attestations/PADO.md:2:8 - Unknown word (PADO)
/home/runner/work/doc.zk-evm/doc.zk-evm/docs/build-on-linea/tooling/attestations/PADO.md:5:4 - Unknown word (PADO)
/home/runner/work/doc.zk-evm/doc.zk-evm/docs/build-on-linea/tooling/attestations/PADO.md:7:2 - Unknown word (PADO)
/home/runner/work/doc.zk-evm/doc.zk-evm/docs/build-on-linea/tooling/attestations/PADO.md:7:124 - Unknown word (PADO)
/home/runner/work/doc.zk-evm/doc.zk-evm/docs/build-on-linea/tooling/attestations/PADO.md:9:29 - Unknown word (PADO)
/home/runner/work/doc.zk-evm/doc.zk-evm/docs/build-on-linea/tooling/attestations/PADO.md:21:110 - Unknown word (PADO's)
/home/runner/work/doc.zk-evm/doc.zk-evm/docs/build-on-linea/tooling/attestations/PADO.md:26:54 - Unknown word (Verax)
/home/runner/work/doc.zk-evm/doc.zk-evm/docs/build-on-linea/tooling/attestations/PADO.md:26:283 - Unknown word (Verax)

**Comment by xiafubiao** [comment]
I created a new PR in the [new link](https://github.com/Consensys/doc.zk-evm/pull/328)

File: package-lock.json
[the remark this answers is not in the document]
Comment (xiafubiao) [reply package-lock.json:1]: Hey Emily, thx for the review, I fixed the issues (the package-lock.json issue, and the lint pass issue) and have now created a new PR in another [thread](https://github.com/Consensys/doc.zk-evm/pull/328). sorry for the late reply due to the holidays

**Pull request closed without merging**